src/components/color-picker/color-picker.scss

```diff
@@ -143,6 +143,10 @@ $gap--large: 12px;
   flex-wrap: wrap;
 }
 
+.channels--rtl {
```

Comment:
Wouldn't it be simpler to fix the direction of `.channels` since it should follow the same order regardless of page direction?

Reply:
fixing direction of `.channels` wouldn't work with the logical props.

Reply:
Couldn't you just set the `direction` to always be `ltr`? Then you don't have to do a dom query to get the direction.

Reply:
setting the direction to `ltr` would make the `calcite-input` prefix to be `ltr` always .

Reply:
@anveshmekala will unavailable for a while, so I've gone ahead and pushed a revisited approach. @driskull Can you review?